bindings_test.go

```diff
@@ -1151,6 +1167,20 @@ func testLOBRetrieval(t *testing.T, useArrowFormat bool) {
 	})
 }
 
+func TestMaxLobSizeSwitch(t *testing.T) {
+	runDBTest(t, func(dbt *DBTest) {
+		dbt.exec(disableLargeVarcharAndBinary)
```

Comment:
What if we run in an env:
1. where this parameter is disabled by default?
2. where this parameter is not available?

Reply:
Then we can use dbt.mustExec and it will throw error when this occurs :) Changed this.